trojsten/utils/utils.py

```diff
@@ -2,8 +2,58 @@
 from datetime import time
 from decimal import Decimal
 
+from django.conf import settings
 from django.http import HttpResponse
 from django.utils import timezone
+from django.utils.translation import ngettext
+
+
+def get_progressbar_data(start, end):
```

Comment:
Tieto veci by mohli byt v utils.progressbar, nie utils.utils aby to nebol uplny bordel.